java/api/src/main/java/io/ray/api/utils/parallelactor/ParallelCall.java

```diff
@@ -0,0 +1,666 @@
+// Generated by `RayCallGenerator.java`. DO NOT EDIT.
+
+package io.ray.api.utils.parallelactor;
+
+import io.ray.api.utils.parallelactor.ParallelActor;
+import io.ray.api.utils.parallelactor.ParallelActorCreator;
+import io.ray.api.function.PyFunction;
+import io.ray.api.ObjectRef;
+import io.ray.api.function.RayFunc0;
+import io.ray.api.function.RayFunc1;
+import io.ray.api.function.RayFunc2;
+import io.ray.api.function.RayFunc3;
+import io.ray.api.function.RayFunc4;
+import io.ray.api.function.RayFunc5;
+import io.ray.api.function.RayFunc6;
+import io.ray.api.function.RayFuncVoid0;
+import io.ray.api.function.RayFuncVoid1;
+import io.ray.api.function.RayFuncVoid2;
+import io.ray.api.function.RayFuncVoid3;
+import io.ray.api.function.RayFuncVoid4;
+import io.ray.api.function.RayFuncVoid5;
+import io.ray.api.function.RayFuncVoid6;
+
+/**
+ * This class provides type-safe interfaces for `Ray.call` and `Ray.createActor`.
+ **/
+class ParallelCall {
+  // ===========================
+  // Methods for actor creation.
+  // ===========================
+  public static <A> ParallelActorCreator<A> parallelActor(RayFunc0<A> f) {
```

Comment:
I think `actor` is better than `parallelActor` because parallel is mentioned in the package name.